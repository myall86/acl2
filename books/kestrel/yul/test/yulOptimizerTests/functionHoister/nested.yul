{
    let a
    function f() {
        let b
        function g() { let c }
        let d
    }
}
// ====
// dialect: yul
// ----
// step: functionHoister
//
// {
//     let a
//     function g()
//     { let c }
//     function f()
//     {
//         let b
//         let d
//     }
// }
