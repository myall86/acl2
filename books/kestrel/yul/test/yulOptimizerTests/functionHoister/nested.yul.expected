
 {
     let a
     function g()
     { let c }
     function f()
     {
         let b
         let d
     }
 }
